library/kani/src/lib.rs

```diff
@@ -120,15 +120,15 @@ pub fn any<T: Arbitrary>() -> T {
 /// under all possible `NonZeroU8` input values between 0 and 12.
 ///
 /// ```rust
-/// let inputA = kani::any_where::<std::num::NonZeroU8>(|x| *x < 12, "explanation");
+/// let inputA = kani::any_where::<std::num::NonZeroU8>(|x| *x < 12);
```

Comment:
Can you please remove the line `/// The explanation field gives a mechanism to explain why the assumption is required for the proof.` that appears above?

Reply:
Good catch @adpaco-aws! Removed.